prep-html-syntax/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Syntax</title>
+  </head>
+  <body>
+
+    <h1 style="color: purple;">My favorite Star Wars character is Mace Windu!</h1>
+
+    <img src="https://static.wikia.nocookie.net/starwars/images/f/fc/Mace_Windu.jpg/revision/latest?cb=20071230055326" alt="Mace Windu" height="500" width="400">
+    <br><br>
+    <p1 style="color: darkgreen;">Mace Windu is my favorite Star Wars character because he is Samuel L. Jackson and has a purple lightsaber.</p1>
```

Comment:
Hey Daniel, heads up, `p1` is not a valid HTML element, use a `<p></p>` instead.

Reply:
You're right! Thank you!